src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.cs

```diff
@@ -2,14 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Win32.SafeHandles;
 using System.Security.AccessControl;
-using System;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.IO
 {
-    public static class FileSystemAclExtensions
+    public static partial class FileSystemAclExtensions
     {
+
```

Comment:
Nit: why was a blank line added here?

Reply:
It was a leftover from when I split this file.